README.md

```diff
@@ -228,6 +259,7 @@ Services to securely store your Docker images.
 * [VMWare Harbor](http://vmware.github.io/harbor/) Project Harbor by VMWare is an enterprise-class registry server that stores and distributes Docker images. Harbor extends the open source Docker Distribution by adding the functionalities usually required by an enterprise, such as security, identity and management.
 
 ## Image Builder
+* [Building good docker images ![doc]](http://jonathan.bergknoff.com/journal/building-good-docker-images) by [@jbergknoff](https://github.com/jbergknoff)
```

Comment:
Mixing articles with project will lead to a real mess! So no, nobody should do that.